### PR TITLE
PLUGIN-905-pom, feature, runner and stepdef

### PR DESCRIPTION
**Pull request by ksahilCF**

### CODE REVIEW

**Comment by ksahilCF** [comment]
@tivv @rmstar please review the changes.

File: src/e2e-test/stepsDesign/GCSBasicDemo.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+import io.cucumber.java.en.When;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.UUID;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org. openqa.selenium.support.ui.Select;
```
Comment (rmstar) [inline src/e2e-test/stepsDesign/GCSBasicDemo.java:15]: remove space

File: pom.xml [hunk 2]
```diff
@@ -793,5 +793,17 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>cloudBuilde2e</id>
```
Comment (tivv) [inline pom.xml]: I don't think it's dependent on cloud build. Name it `e2e-tests`

File: pom.xml [hunk 3]
```diff
@@ -793,5 +793,17 @@
...
+      <dependencies>
+              <dependency>
+                <groupId>io.cdap.tests.e2e</groupId>
+                <artifactId>cdap-e2e-framework</artifactId>
+                <version>0.0.1-SNAPSHOT</version>
+                <scope>compile</scope>
```
Comment (tivv) [inline pom.xml]: Do you need it for `compile`? `test` scope should be enough

**Comment by tivv** [comment]
Please add JIRA number to the PR title as well

**Review by rmstar — COMMENTED** [review]
Please reformat PR to conform to Cask style

**Comment by rmstar** [comment]
https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard

File: pom.xml (hunk 2 above)
Comment (ksahilCF) [reply pom.xml]: changed name to e2e-tests

File: pom.xml (hunk 3 above)
Comment (ksahilCF) [reply pom.xml]: Changed scope to test

File: src/e2e-test/stepsDesign/GCSBasicDemo.java (hunk 1 above)
Comment (ksahilCF) [reply src/e2e-test/stepsDesign/GCSBasicDemo.java:15]: removed space

**Review by tivv — APPROVED** [review]

**Pull request merged**